### PR TITLE
First commit from Madiyar. Added teacher service

### PR DESCRIPTION
**Pull request by madiyard**
Couldn't find Gadir and Natalie to review

### CODE REVIEW

File: src/main/java/com/cydeo/service/TeacherService.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    @Override
+    public List findAll() {
+        return new ArrayList(Database.teacherList);
+    }
+
+    @Override
+    public void save(Teacher teacher) {
+
```
Comment (Cemil5) [inline src/main/java/com/cydeo/service/TeacherService.java:25]: Please finish this method

File: src/main/java/com/cydeo/service/TeacherService.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    @Override
+    public void update(Teacher teacher) {
```
Comment (Cemil5) [inline src/main/java/com/cydeo/service/TeacherService.java:29]: Please finish this method

File: src/main/java/com/cydeo/service/TeacherService.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    }
+
+    @Override
+    public void deleteById(int id) {
+
```
Comment (Cemil5) [inline src/main/java/com/cydeo/service/TeacherService.java:35]: Please finish this method

**Review by Cemil5 — CHANGES_REQUESTED** [review]
Please finish last three methods

**Review by gibrahimov — CHANGES_REQUESTED** [review]
we are still waiting for methods got updated.

**Review by Cemil5 — APPROVED** [review]

**Review by Cemil5 — APPROVED** [review]

**Review by khashayarshayesteh — APPROVED** [review]

**Review by gibrahimov — APPROVED** [review]
((

**Pull request merged**